artifacttypes/tosca.artifacts/installPipeline/installPipeline.tosca

```diff
@@ -0,0 +1,8 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+artifact_types:
+  installPipeline:
+    metadata:
+      targetNamespace: tosca.artifacts
```

Comment:
Pls avoid using the `tosca.*` namesapce

Reply:
Moved it to radon.artifacts.datapipeline